modules/creating-oci-resources-services.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_oci/installing-oci-assisted-installer.adoc [Using the Assisted Installer to install a cluster on OCI]
+
+:_mod-docs-content-type: PROCEDURE
+[id="creating-oci-resources-services_{context}"]
+= Creating OCI resources and services
+
+Create {oci-first} resources and services so that you can establish infrastructure with governance standards that meets your organization requirements.
+
+By creating a compartment, you can better organize, restrict access, and set usage limits to {oci} resources. Additionally, by creating an object storage bucket, you can safely and securely store and retrieve data from your cloud environment.
+
+.Prerequisites
+
+* You configured an {oci} account to host the cluster. See link:https://docs.oracle.com/en/cloud/get-started/subscriptions-cloud/csgsg/create-users-and-assign-roles.html#GUID-1041AC78-9E43-42CF-B5A5-73F8B8FEB39A[Create Users and Assign Roles] in the Oracle documentation.
+* You set up your cloud infrastructure, such as compute, networking, and storage settings, before installing {product-title}. You must ensure that your infrastructure meets the minimum Assisted Installer link:https://access.redhat.com/documentation/en-us/assisted_installer_for_openshift_container_platform/2024/html-single/installing_openshift_container_platform_with_the_assisted_installer/index#hardware[Hardware]. 
+* If the cloud identity and access management (IAM) APIs are not accessible in your environment, or if you do not want to store an administrator-level credential secret in the `kube-system` namespace, you can manually create and maintain IAM credentials. See link:https://docs.oracle.com/en-us/iaas/Content/Identity/Concepts/overview.htm[Overview of Identity and Access Management] in the Oracle documentation.
+
+.Procedure
+
+. Log in to your link:https://cloud.oracle.com/a/[Oracle Cloud Infrastructure] account with administrator privileges.
+
+. Create a child compartment. See link:https://docs.public.oneportal.content.oci.oraclecloud.com/en-us/iaas/process-automation/oci-process-automation/create-compartment.html[Create a compartment] in the Oracle documentation.
++
+[NOTE]
+====
+When creating the child compartment, specify the default parent compartment or any other parent compartment from the list.
+====
+
+. Record the name and the link:https://docs.oracle.com/en-us/iaas/Content/libraries/glossary/ocid.htm[{ocid-first}] of the compartment
+
+. Create a bucket resource for your child compartment. Ensure that you specify your child compartment in the **Create in compartment** field for the bucket resource. 
+
+. Go to **Object Storage & Archive Storage > Buckets** and create a bucket, where **Bucket name** refers to your cluster's name.
```

Comment:
Not sure if we should explain in details but at least it should be marked as Prerequisites creation of OCI resources as:
- VCNs and network security groups
- Load-balancers
- DNS
- Instances
- Storage buckets and create custom images
- Identities and policies

Reply:
Yes